### PR TITLE
Shell stuff

### PR DESCRIPTION
**Pull request by stevenewsted**

### CODE REVIEW

File: install-oc-tools.sh [hunk 1]
```diff
@@ -364,10 +369,10 @@ remove_old_ver() {
 
   if ls /usr/local/bin/oc*bak 1> /dev/null 2>&1 && ls /usr/local/bin/openshift-install*bak 1> /dev/null 2>&1 && ls /usr/local/bin/kubectl*bak 1> /dev/null 2>&1
   then
-  read -p "Delete the following files?
+  read -rp "Delete the following files?"
```
Comment (cptmorgan-rh) [inline install-oc-tools.sh]: Steve, 

In the original code lines [L367-370](https://github.com/cptmorgan-rh/install-oc-tools/blob/4e1e8a78fee5121668df771679b245b2dfd6723e/install-oc-tools#L367-L370) are in one set of quotation marks for the `for loop` to work.

Output of original code:

~~~
sudo install-oc-tools --cleanup
Delete the following files?

/usr/local/bin/oc.4.3.25.bak
/usr/local/bin/oc.4.4.6.bak
/usr/local/bin/oc.4.4.8.bak
/usr/local/bin/oc.4.6.0-0.nightly-2020-06-20-011219.bak
/usr/local/bin/oc.openshift-clients-4.2.32-202005020632.bak
/usr/local/bin/kubectl.4.3.25.bak
/usr/local/bin/kubectl.4.4.6.bak
/usr/local/bin/kubectl.4.4.8.bak
/usr/local/bin/kubectl.4.6.0-0.nightly-2020-06-20-011219.bak
/usr/local/bin/kubectl.openshift-clients-4.2.32-202005020632.bak
/usr/local/bin/openshift-install.4.3.25.bak
/usr/local/bin/openshift-install.4.4.6.bak
/usr/local/bin/openshift-install.4.4.8.bak
/usr/local/bin/openshift-install.4.6.0-0.nightly-2020-06-20-011219.bak
/usr/local/bin/openshift-install.openshift-clients-4.2.32-202005020632.bak

Y/N?
~~~

Your PR:
~~~
sudo bash install-oc-tools.sh --cleanup
Delete the following files?n
Error: unknown command "/usr/local/bin/oc.4.4.6.bak" for "oc"
Run 'oc --help' for usage.
~~~

Everything else looks good.

**Review by cptmorgan-rh — CHANGES_REQUESTED** [review]
Fix quotations on Line [372-375](https://github.com/cptmorgan-rh/install-oc-tools/blob/4d25c10d66708630c95b50da4eff82bfc15274a9/install-oc-tools.sh#L372-L375) to restore cleanup functionality.

File: install-oc-tools.sh (hunk 1 above)
Comment (stevenewsted) [reply install-oc-tools.sh]:
```
-> % sudo ./install-oc-tools.sh --cleanup
Delete the following files?

/usr/local/bin/oc.4.4.8.bak
/usr/local/bin/kubectl.4.4.8.bak
/usr/local/bin/openshift-install.4.4.8.bak

Y/N? y

```
All better, good catch!

**Review by cptmorgan-rh — APPROVED** [review]
Looks good.

**Pull request merged**